dotcom-rendering/src/web/components/BlockquoteBlockComponent.tsx

```diff
@@ -1,10 +1,10 @@
-import { css } from '@emotion/react';
+import { css, jsx as h } from '@emotion/react';
```

Comment:
Why rename from `jsx` to `h`? We go to great lengths elsewhere to abstract away the fact that this repo uses Preact.